lib/board.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :value_board
+
+  def initialize
+    @value_board = %w[1 2 3 4 5 6 7 8 9]
+  end
+
+  def display_board
+    puts " #{value_board[0]} | #{value_board[1]} | #{value_board[2]}\n___|___|___\n #{value_board[3]} | #{value_board[4]} | #{value_board[5]}\n___|___|___\n #{value_board[6]} | #{value_board[7]} | #{value_board[8]}"
+  end
+end
+
+x = Board.new
```

Comment:
Since there's a "runner" now, I don't think the board class file needs to print out the board. As a bonus this will clean up the test output. As it is, the board gets printed out along with the little RSpec dots. It doesn't affect functionality, of course, but it's something you may want to clean up.